mob.go

```diff
@@ -702,7 +702,7 @@ func execute(command string, parameter []string, configuration Configuration) {
 		} else if configuration.Timer != "" {
 			startTimer(configuration.Timer, configuration)
 		} else {
-			sayInfo("It's now " + currentTime() + ". Happy collaborating!")
+			sayInfo("It's now " + currentTime() + ". Happy collaborating! :)")
```

Comment:
Should we use the Unicode one?
https://emojipedia.org/smiling-face-with-smiling-eyes/

Reply:
I kind of like the subtle nature of ":)" but open to using a Unicode one too :)

Reply:
I would go with the subtle nature for now.